extensions/arc/src/ui/dashboards/postgres/postgresDashboardPage.ts

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import { ControllerModel } from '../../../models/controllerModel';
+import { PostgresModel } from '../../../models/postgresModel';
+import { DashboardPage } from '../../components/dashboardPage';
+
+export abstract class PostgresDashboardPage extends DashboardPage {
```

Comment:
This class isn't really doing anything - unless you plan on adding something  why not just have the specific dashboard pages extend DashboardPage directly?

Reply:
It gives all the Postgres pages access to the PostgresModel and ControllerModel.  Rather than define this constructor on all the Postgres pages.